WebDevStudios/wd_s
# Move i18n build to composer, fix language path

Closes #566 and #595 

### DESCRIPTION

- Moved the i18n make-pot action from npm to composer as it requires wp-cli & php.

### OTHER


### STEPS TO VERIFY

Run a `npm run build` normally, it should do the same thing.
Run `npm run theme:node` and it should only build the npm related stuff.

### DOCUMENTATION

Will this pull request require updating the wd_s [wiki](https://github.com/WebDevStudios/wd_s/wiki)?
Possibly?
